_data/components/linebarchart.yml

```diff
@@ -0,0 +1,87 @@
+directive : "o-chart"
+chart: true
+
+inheritedAttributes: [{
+  component: "OChartComponent",
+  path: "charts/overview",
```

Comment:
Incorrect path. It would be _"charts/configuration/overview"_
Please review the other configuration files.